### PR TITLE
4422_windowItemDisabled

### PR DESCRIPTION
**Pull request by MayaElf**
Feature Windows item 'disabled' is added

### CODE REVIEW

File: components/groups/windows/DisabledWindowItem.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              class="white--text"
+            >
+              Slide {{ n }}
+            </h1>
+          </v-row>
+        </v-card>
+      </v-window-item>
+    </v-window>
+
+    <v-card-actions class="justify-space-between">
```
Comment (omega790) [inline components/groups/windows/DisabledWindowItem.vue:35]: Please delete this part as this actions don't work with v-window-item properties

File: components/groups/windows/DisabledWindowItem.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        @click="next"
+      >
+        <v-icon>mdi-chevron-right</v-icon>
+      </v-btn>
+    </v-card-actions>
+  </v-card>
+</template>
+<script>
+export default {
+  data: () => ({
```
Comment (omega790) [inline components/groups/windows/DisabledWindowItem.vue:72]: And as we don't use custom navigation, please delete script for it

File: components/groups/windows/DisabledWindowItem.vue [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<template>
+  <v-card
+    flat
+    tile
+  >
+    <v-window
+      v-model="onboarding"
+      show-arrows
+    >
+      <v-window-item
```
Comment (omega790) [inline components/groups/windows/DisabledWindowItem.vue:10]: What do you think about adding items not in loop and, for example, add 3 items with different text and make one of them disabled (so we can see that when we are using navigation arrows we'll skip disabled item)

**Review by omega790 — CHANGES_REQUESTED** [review]

File: components/groups/windows/DisabledWindowItem.vue (hunk 1 above)
Comment (MayaElf) [reply components/groups/windows/DisabledWindowItem.vue:35]: DONE

File: components/groups/windows/DisabledWindowItem.vue (hunk 2 above)
Comment (MayaElf) [reply components/groups/windows/DisabledWindowItem.vue]: DONE

File: components/groups/windows/DisabledWindowItem.vue (hunk 3 above)
Comment (MayaElf) [reply components/groups/windows/DisabledWindowItem.vue:9]: DONE

**Review by omega790 — APPROVED** [review]

**Pull request merged**